### PR TITLE
Fix targdiff

### PR DESCRIPTION
**Pull request by alexeilebedev**

### CODE REVIEW

**Comment by vparizhs** [comment]
both master and fix-targdiff branch look broken. Just running it in either branch gives these errors... No use instructions are given so I don't really know how it supposed to behave and what it does... Please add an example...

$ targ-diff
bin/targ-diff: 8: [: unexpected operator
bin/targ-diff: 14: [: 0: unexpected operator
Cloning into '.tempgit'...
done.
fatal: empty string is not a valid pathspec. please use . instead if you meant to match all paths

**Review by vparizhs — CHANGES_REQUESTED** [review]
1. the utility produces errors when run - see below.
2. Please add a use example - it is hard to see what it is supposed to do...

$ targ-diff
bin/targ-diff: 8: [: unexpected operator
bin/targ-diff: 14: [: 0: unexpected operator
Cloning into '.tempgit'...
done.
fatal: empty string is not a valid pathspec. please use . instead if you meant to match all paths

File: bin/targ-diff [hunk 1]
```diff
@@ -3,8 +3,8 @@
 # between current working directory and specified commit ($2, default:HEAD~)
 # detecting movement across files
 
-BRANCH=$(git-current-branch)
```
Comment (polovinkin-alexey) [inline bin/targ-diff:6]: I  guess  this command or executable is missing on the repo.

**Comment by polovinkin-alexey** [comment]
PR title is irrelevant to the majority of changes.

**Comment by alexeilebedev** [comment]
Closed for now, will reimplement targdiff in a different way

**Pull request closed without merging**